### PR TITLE
New export image dialog

### PR DESCRIPTION
**Pull request by javierarce**
Replaces the old export image dialog with the new version.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5325/))

**Comment by saleiva** [comment]
Can we give this another try before releasing?

**Comment by javierarce** [comment]
@saleiva: @josecruz tested it yesterday and found a bug. Do you want to try this too? I can upload it to staging again if it's needed.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5359/))

File: lib/assets/javascripts/cartodb/table/views/export_image_view.jst.ejs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <div class="HelperEast"></div>
+  <div class='CanvasExport'></div>
+
+  <div class="ExportImageView-Footer">
+
+    <div class="ExportImageView-Footer--inner">
+
+      <div class="ExportImageView--dimensions">
+        <span class="dimension">w:</span> <strong class="js-width"><%= width %></strong> <span class="unit">px</span>,
+        <span class="dimension">h:</span> <strong class="js-height"><%= height %></strong> <span class="unit">px</span>
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/table/views/export_image_view.jst.ejs]: <%- make it a habit ;)

**Comment by saleiva** [comment]
Remember we should do a great QA session on this. Where is it deployed?

**Comment by viddo** [comment]
(rrerequisite to remove old modal code #4673, just leaving a reference so I can see on that issue that this PR is actually merged before I start working on it)

**Comment by javierarce** [comment]
Sorry, I'm moving this PR to https://github.com/CartoDB/cartodb/pull/4809. I'll take care to remove the old code, @viddo.

**Pull request closed without merging**